docs/getting-started/tools/index.md

```diff
@@ -39,8 +39,7 @@ environment.
 - [NPM](https://www.npmjs.com/) v8 (included with Node)
 - [Rust](https://www.rust-lang.org/tools/install) - Used for native desktop components
 - [Git](https://git-scm.com)
-  - [Commit signing](../../contributing/commit-signing.mdx) is required for all _Bitwarden
-    contributors_ and is strongly encouraged for _community contributors_.
+  - [Commit signing](../../contributing/commit-signing.mdx) is strongly recommended
```

Comment:
❓ Spacing issue?

Reply:
I think it's meant to be indented under the "Git" dot point (given that it's about configuring git), is that what you mean?

Reply:
Yes, so if that's intentional we are good!